code/testapp/src/main/java/com/adobe/edge/media/testapp/analytics/MediaAnalyticsProvider.java

```diff
@@ -9,15 +9,19 @@
   governing permissions and limitations under the License.
 */
 
-package com.adobe.mediaanalyticstestapp.analytics;
+package com.adobe.edge.media.testapp.analytics;
```

Comment:
let's rename analytics package to either 
`tracking` || `handler` || `api.implementation`

Reply:
Let's rename the file to 
EdgeMediaAPIImplementation
EdgeMediaAPICaller
EdgeMediaTracker

Reply:
What about renaming the package to `tracker` and the class to `MediaPlayerObserver` as the class implements the `Observer` interface?

Reply:
Sounds good to me @kevinlind.